### PR TITLE
LPS-122211 set default value if it does not exist since app builder remains the value through the locales

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-122211

### CODE REVIEW

**Comment by marcosapmf** [comment]
cc/ @interaminense @kevenleone

**Comment by kevenleone** [comment]
Just started reviewing :)

**Review by kevenleone — CHANGES_REQUESTED** [review]
The rest for me is fine, @marcosapmf, do you agree?

Thanks for let us know.

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/FieldPreview.es.js [hunk 1]
```diff
@@ -310,7 +309,7 @@ export const FieldValuePreview = ({
 		.filter((key) => key.includes(fieldName))
 		.map((key) => {
 			if (typeof dataRecordValues[key] == 'object') {
-				return dataRecordValues[key][defaultLanguageId];
```
[the remark this answers is not in the document]
Comment (marcosapmf) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/FieldPreview.es.js:313]: Hey @kevenleone,
thanks a lot for reviewing this pull.

I agree with you, so I'm going to make this change and make a push force.

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 , 

Good job !

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Hey, @marcosapmf. Apparently, the failed test is related to this pull. Could you please check that?
Let me know if you need any help. :)

**Comment by marcosapmf** [comment]
resent https://github.com/liferay-data-engine/liferay-portal/pull/316

**Pull request closed without merging**